lib/devices/android/bootstrap/src/io/appium/uiautomator/core/InteractionController.java

```diff
@@ -0,0 +1,49 @@
+package io.appium.uiautomator.core;
+
+import android.view.InputEvent;
+import android.view.MotionEvent;
+import android.view.MotionEvent.PointerCoords;
+
+import static io.appium.android.bootstrap.utils.ReflectionUtils.invoke;
+import static io.appium.android.bootstrap.utils.ReflectionUtils.method;
+
+public class InteractionController {
+
+  private static final String CLASS_INTERACTION_CONTROLLER = "com.android.uiautomator.core.InteractionController";
+  private static final String METHOD_SEND_KEY = "sendKey";
+  private static final String METHOD_INJECT_EVENT_SYNC = "injectEventSync";
+  private static final String METHOD_TOUCH_DOWN = "touchDown";
+  private static final String METHOD_TOUCH_UP = "touchUp";
+  private static final String METHOD_TOUCH_MOVE = "touchDown";
+  public static final String METHOD_PERFORM_MULTI_POINTER_GESTURE = "performMultiPointerGesture";
+
+  private final Object interactionController;
+
+  public InteractionController(Object interactionController) {
+    this.interactionController = interactionController;
+  }
+
+  public boolean sendKey(int keyCode, int metaState){
+    return (Boolean) invoke(method(CLASS_INTERACTION_CONTROLLER, METHOD_SEND_KEY, int.class, int.class), interactionController, keyCode, metaState);
```

Comment:
If you're going to go to the trouble of creating a wrapper, can't you just define all your reflected methods in the ctor rather than repeatedly doing so for each invocation? Or is there a reason not to? It's not a huge deal either way tho.